docs/sources/developers/http_api/auth.md

```diff
@@ -19,72 +19,15 @@ title: 'Authentication HTTP API '
 
 # Authentication API
 
-> If you are running Grafana Enterprise, for some endpoints you would need to have relevant permissions. Refer to [Role-based access control permissions]({{< relref "../../administration/roles-and-permissions/access-control/custom-role-actions-scopes/" >}}) for more information.
-
-## Tokens
-
-Currently, you can authenticate via an `API Token` or via a `Session cookie` (acquired using regular login or OAuth).
-
-## X-Grafana-Org-Id Header
-
-**X-Grafana-Org-Id** is an optional property that specifies the organization to which the action is applied. If it is not set, the created key belongs to the current context org. Use this header in all requests except those regarding admin.
-
-**Example Request**:
-
-```http
-POST /api/auth/keys HTTP/1.1
-Accept: application/json
-Content-Type: application/json
-X-Grafana-Org-Id: 2
-Authorization: Bearer eyJrIjoiT0tTcG1pUlY2RnVKZTFVaDFsNFZXdE9ZWmNrMkZYbk
-
-{
-  "name": "mykey",
-  "role": "Admin",
-  "secondsToLive": 86400
-}
-```
+The Authentication HTTP API is used to manage API keys.
 
-## Basic Auth
+{{% admonition type="note" %}}
+If you use Grafana v9.1 or newer, use service accounts instead of API keys. For more information, refer to [Grafana service account API reference]({{< relref "./serviceaccount/" >}}).
+{{% /admonition %}}
```

Comment:
nit
```suggestion
{{% admonition type="note" %}}
If you use Grafana v9.1, we encourage the usage of Service Accounts over API keys. API keys have been deprecated and will be removed shortly. For more information, refer to [Grafana Service Accounts reference]({{< relref "./serviceaccount/" >}}).
{{% /admonition %}}
```

Reply:
I've copied this note over from another API key page, so might leave it as it is for consistency :)